tools/board.py

```
@@ -0,0 +1,910 @@
+    which silently clears every item field value pointing at the old id. The
+    schema says so in as many words: "Include this to preserve the option's
+    identity during updates, preventing item field values from being cleared."
+
+    So an addition sends every existing option WITH its id, and only the new
+    ones without. Sending names alone is how this field gets wiped, and it
+    fails silently: the write succeeds, the options look right, and every
+    value that referenced them is gone.
+    """
+    parts = [
+        '{id:"%s",name:"%s",color:%s,description:"%s"}'
+        % (o["id"], option_text(o["name"]), option_color(o), option_text(o.get("description")))
```

Ruff (0.16.3)

[warning] Use format specifiers instead of percent format

Replace with format specifiers

(UP031)